response_operations_ui/templates/create-ce-event.html

```diff
@@ -28,17 +28,25 @@ <h1 class="u-fs-xl">Create business event</h1>
 
     <h2 class="u-fs-l u-mt-l">{{event_name}}</h2>
 
-    {% if errors %}
-        <div class="panel panel--simple panel--error u-mb-l">
-          <div class="panel__body" id="save-error">
-            <h2 id='error-saving-message' class="u-fs-r--b">Error creating {{event_name}} date</h2>
-    {% endif %}
+    {%- with errors = get_flashed_messages(category_filter=["error"]) -%}
+        {%- if errors -%}
+            <div class="panel panel--simple panel--error u-mb-l">
+              <div class="panel__body" id="save-error">
+                <h2 id='error-saving-message' class="venus">Error updating {{event_name}} date </h2>
+                <div id='error-text'>
+                  {%- for message in errors -%}
+                  <p>{{ message }}</p>
+                  {%- endfor -%}
+                </div>
+              </div>
+        {%- endif -%}
+    {%- endwith -%}
 
 
     {% if date_restriction_text %}
       {% for text in date_restriction_text %}
       <div>
-        {{ text }}
+       {{ text }}
```

Comment:
Is the date restriction text not another form of error message?  Could this be bundled in with the errors above? If not, this should have `<p>` tags around it.

Reply:
No it's the text that shows on the form always (Must be after blah and before blah), but it can be blank with a fresh collection exercise, <p> tags can go in